tests/spec/fs.stat.spec.js

```diff
@@ -91,4 +91,32 @@ describe('fs.stat', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat).to.be.a('function');
+  });
+  
+  it('should return a promise', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat()).to.be.a('Promise');
+  });
+  
+  it('(promise) should return a stat object if file exists', function(){
+    var fs = util.fs();
+
+    return fs.promises.stat('/')
+      .then(result => { 
+        expect(result).to.exist;
+        expect(result['node']).to.be.a('string');
+        expect(result['dev']).to.equal(fs.name);
+        expect(result['size']).to.be.a('number');
+        expect(result['nlinks']).to.be.a('number');
+        expect(result['atime']).to.be.a('number');
+        expect(result['mtime']).to.be.a('number');
+        expect(result['ctime']).to.be.a('number');
+        expect(result['type']).to.equal('DIRECTORY');
+      })
+      .catch(error => expect(error).not.to.exist);
```

Comment:
I don't think this is necessary.  Unless you are expecting an `error` case, which you aren't here, you can just leave the `catch` off and let the test handle it: since you return your `Promise` on line 108, the encapsulating test (`it()` function) will receive the error and report a failure.